quantulum3/parser.py

```diff
@@ -417,6 +434,53 @@ def clean_text(text, lang="en_US"):
     return text
 
 
+###############################################################################
+def extract_range_ands(text, lang="en_US"):
+    return _get_parser(lang).extract_range_ands(text)
+
+
+###############################################################################
+def handle_consecutive_quantities(quantities, context):
+    """
+    [45] and/or [50 mg] --> add unit to first [45 mg] [50 mg]
+    between [44 mg] and [50 mg] --> range [47+/-3 mg]
+    from [44 mg] to [50 mg] --> same
+    !from [44 mg] to [50 mg] --> range [47+/-3 mg]
```

Comment:
What does the exclamation mark stand for here? Or is this a duplicated comment?

It seems inconsistent with the a ove testcase where "from 44 mg to 50 mg" is parsed to a non-range result

Reply:
this is supposed to indicate that "X to Y" is a range, while "from X to Y" is not a range.

Reply:
Is it just me or are sentences with "from X to Y" not also expressing ranges? Example: "temperatures ranged from −18 °C to −34 °C". Do you have a counterexample? (genuinely interested!)
Found my example in https://en.wikipedia.org/wiki/Dash#Ranges_of_values